packages/react-tinacms-inline/src/styles/focus-ring.tsx

```diff
@@ -84,6 +84,7 @@ export const StyledFocusRing = styled.div<StyledFocusRingProps>(p => {
     position: relative;
     width: 100%;
     height: 100%;
+    min-height: var(--tina-font-size-0);
```

Comment:
```suggestion
    min-height: var(--tina-padding-small);
```

Reply:
Those variables are the same value: https://github.com/tinacms/tinacms/blob/master/packages/%40tinacms/styles/src/Styles.tsx#L61

IMO font-size is more appropriate than padding here, since we're defining the height of a content area (and not its surrounding padding.) Ideally a line-height variable would be most correct, but we don't have that